loom/common/utils/utTypes.h

```diff
@@ -1393,7 +1393,7 @@ class utHashTable
 
 
     // Find and cache key
-    Value *get(const Key& key)
+    Value *get(const Key& key) const
```

Comment:
If it wasn't appropriate, the compiler would complain :) `get` doesn't change any internal variables, and the rest of called methods are also const.